modules/rest_api/controllers/services/rest.php

```diff
@@ -1585,6 +1649,44 @@ private function esGetSpecialFieldLon(array $doc) {
     return "$lon$ew";
   }
 
+  /**
```

Comment:
I'd also recently committed the following: https://github.com/Indicia-Team/warehouse/commit/c15541463c2f271c85dc6d996274777470899912#diff-403d824cd34240a1224f8fc4faf3401e. The formats are slightly different though - do you think this warrants multiple special fields, or perhaps a single field with a format option? E.g. #lat:decimal# or #lat:nssuffix# or something like that? I don't have a strong view - just want to make sure this is an active decision rather than something we didn't think through.

Reply:
I prefer the your suggestion of a format option so that the functionally similar code is kept together and more compact.

Reply:
Are you happy to code this change?

Reply:
@johnvanbreda - I must literally have been doing it whilst you typed. It's done now.